### ISSUE TITLE
Feature request: be an option when right-clicking in a textarea

### ISSUE THREAD

**Issue by pjz** [jlebon/textern#48]
To be more discoverable, There should be an 'Edit locally via textern' option in the right-click menu of a textarea.

**Comment by jlebon** [comment]
> To be more discoverable

Though this is an addon one knowingly installs, right? It's not a baked-in Firefox feature.

I'm not strongly against adding a context menu entry, though even convenience-wise, ISTM like it's easier and faster anyway to just left-click and press the shortcut.

**Comment by pjz** [comment]
Fair enough, but my editing in big textareas is intermittent, so i forget the hotkey so end up having to go find the extension configuration to look it up.

**Comment by yjfguhd** [comment]
There are some sites that annoyingly change, or seem to disable altogether, keyboard shortcuts. A menu entry would be useful in those cases.
